modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -6,9 +6,9 @@
 [id="persistent-storage-csi-vsphere-install-issues_{context}"]
 = Removing a third-party vSphere CSI Operator Driver
 
-{product-title} 4.10 includes a built-in version of the vSphere Container Storage Interface (CSI) Operator Driver that is supported by Red Hat. If you have installed a vSphere CSI driver provided by the community or another vendor, upgrades will be prevented in a future release of {product-title}.
+{product-title} 4.12, and later, includes a built-in version of the vSphere Container Storage Interface (CSI) Operator Driver that is supported by Red Hat. If you have installed a vSphere CSI driver provided by the community or another vendor, updates to the next major version of {product-title}, such as 4.13, or later, might be disabled for your cluster.
```

Comment:
Technically we do ship the builtin operator from 4.10 itself. So we shouldn't change versioning there I think.

Reply:
yes agree it may confuse customers.

Reply:
Fixed it.